homeassistant/data_entry_flow.py

```diff
@@ -137,7 +138,7 @@ def __init__(
         self.description_placeholders = description_placeholders
 
 
-class FlowResult(TypedDict, total=False):
+class BaseFlowResult(Generic[_HandlerT], TypedDict, total=False):
```

Comment:
```suggestion
class BaseFlowResult(TypedDict, Generic[_HandlerT], total=False):
```
Just a style preference. I like to put `Generic` last.